evd_ros_core/RobotMoveTrajectory goal
---
bool status